.Rbuildignore

```diff
@@ -1,2 +1,4 @@
+^renv$
+^renv\.lock$
```

Comment:
You don't *have* to add this btw, I did it for a couple of modules because I use `renv` locally for basically all my projects now.